awx/main/access.py

```diff
@@ -1094,11 +1096,7 @@ def can_cancel(self, obj):
         return self.user in obj.inventory_source.inventory.admin_role
 
     def can_start(self, obj, validate_license=True):
-        # For relaunching
-        if obj and obj.inventory_source:
-            access = InventorySourceAccess(self.user)
-            return access.can_start(obj.inventory_source, validate_license=validate_license)
-        return False
+        return InventorySourceAccess(self.user).can_start(obj, validate_license=validate_license)
```

Comment:
Meet the new access logic, same as the old access logic. @jbradberry this should do the same thing, but by leveraging the fact that `obj.inventory` is the same as `obj.inventory_source.inventory`. The inventory field on the InventoryUpdate model was recently introduced.